sstables/sstables.hh

```diff
@@ -507,6 +507,10 @@ public:
         sstring prefix() const { return dir; }
     };
 
+    const filesystem_storage& get_storage() const {
```

Comment:
It's better not to, and since it's used only by tests, the `class sstables::test` is the way to put this method into